.githubmap

```diff
@@ -8,15 +8,35 @@
 #
 #
 ajarr Ramana Raja <rraja@redhat.com>
+amitkumar50 Amit Kumar <amitkuma@redhat.com>
+athanatos Samuel Just <sjust@redhat.com>
+badone Brad Hubbard <bhubbard@redhat.com>
+bassamtabbara Bassam Tabbara <bassam.tabbara@quantum.com>
 batrick Patrick Donnelly <pdonnell@redhat.com>
+branch-predictor Piotr Dałek <piotr.dalek@corp.ovh.com>
+chhabaramesh Ramesh Chander <Ramesh.Chander@sandisk.com>
+dvanders Dan van der Ster <daniel.vanderster@cern.ch>
 fullerdj Douglas Fuller <dfuller@redhat.com>
 gregsfortytwo Gregory Farnum <gfarnum@redhat.com>
+ifed01 Igor Fedotov <ifedotov@mirantis.com>
+ivancich J. Eric Ivancich <ivancich@redhat.com>
 jcsp John Spray <john.spray@redhat.com>
 jlayton Jeff Layton <jlayton@redhat.com>
 joscollin Jos Collin <jcollin@redhat.com>
+leseb Sébastien Han <seb@redhat.com>
 liewegas Sage Weil <sage@redhat.com>
+liupan1111 Pan Liu <liupan1111@gmail.com>
+majianpeng Jianpeng Ma <jianpeng.ma@intel.com>
+markhpc Mark Nelson <mnelson@redhat.com>
+rchagam Chagam, Anjaneya <anjaneya.chagam@intel.com>
```

Comment:
This one is different from others: `Lastname, Firstname`. Need to correct it ?

Reply:
Yes, thanks!

Reply:
@batrick I was thinking, do we have to create PRs like this to update `.githubmap` or let the Reviewer or Merging Person do it later during their Review/Merge ? What is the plan ?

Reply:
Personally, I would recommend the reviewer just adding the username to `.githubmap` in a quick single commit to master. Then merge onto that.

For scripts that use `.githubmap` to function, the file must be up-to-date before the merge...

Reply:
@batrick I think, as there is no script at the moment, you can send an email to the ceph-devel informing this plan to everyone.